### ISSUE TITLE
[Improvement] [Accessibility] Allow changing the folding "heading" font style

### ISSUE THREAD

**Issue by sparcbr** [getinsomnia/insomnia#2404]
Is there a way to change this bold/glowing of the purple text that appears inside the { } when a fold is closed in editor? Check the image below..

Explanation:
I don't like this glowing style as it lowers the contrast and makes it harder to see.

![image](https://user-images.githubusercontent.com/5775999/87368042-f5cc0480-c552-11ea-823e-8542c56e0a9c.png)

**Comment by develohpanda** [comment]
This should be possible - either CodeMirror contains a prop we can use to set the placeholder color, or we do some CSS overriding on our side. In any case, I agree, the glow also seems out of place within Insomnia. 💄

Looks strange even in the high-contrast theme

![image](https://user-images.githubusercontent.com/4312346/87369044-9e21ad80-c5d3-11ea-98b1-d97f5a770c4f.png)
